lib/__test__/rule-list.test.js

```diff
@@ -181,12 +188,14 @@ describe('rules', () => {
 
     suite('common denominators', rules.COMMON_DENOMINATOR, [
         ['2/6 + 1/4', '(2 * 2) / (6 * 2) + (1 * 3) / (4 * 3)'],
+        ['2/(3 + x) + 3/(2 + x)', '(2 * (2 + x)) / ((3 + x) * (2 + x)) + (3 * (3 + x)) / ((3 + x) * (2 + x))'],
```

Comment:
There should be some tests with:
- a mix of numeric denominators and variable denominators
- repeated variable denominators
- repeated numeric denominators
- non-fractional terms in the sum